mysite/search/views.py

```diff
@@ -68,7 +68,15 @@ def search_index(request, invalid_subscribe_to_alert_form=None):
 
     format = request.GET.get('format', None)
     start = int(request.GET.get('start', 1))
-    end = int(request.GET.get('end', 10))
+
+    # If user selects number of bugs from drop-down list then 'limit' is the
+    # number of bugs to display obtained by POST otherwise display 10 bugs
+    if request.method == 'POST':
+        limit = int(request.POST.get('items'))
+        end = start + limit - 1
+    else:
+        limit = int(request.GET.get('items', 10))
+        end = int(request.GET.get('end', limit))
```

Comment:
@vipul-sharma20 I'd like to request that your code validate the `limit` so that it's always one of `[10, 20, 30, 40]`. Otherwise I am concerned people might request a `limit=100000` and then use up lots of resources on the server.

Similarly, I'd like to request that you add a unit test for this new functionality.